src/dl_typelib_write_c_header.cpp

```diff
@@ -492,64 +508,67 @@ static void dl_context_write_type( dl_ctx_t ctx, dl_type_storage_t storage, dl_t
+			dl_error_t err = dl_reflect_get_enum_info( ctx, tid, &enum_info );
+			if (DL_ERROR_OK != err) return err;
 			dl_binary_writer_write_string_fmt(writer, "%s", enum_info.name);
-			return;
+		    return DL_ERROR_OK;
 		}
 		case DL_TYPE_STORAGE_ENUM_INT64:
 		case DL_TYPE_STORAGE_ENUM_UINT64:
 		{
 			dl_enum_info_t enum_info;
-			dl_reflect_get_enum_info( ctx, tid, &enum_info );
+			dl_error_t err = dl_reflect_get_enum_info( ctx, tid, &enum_info );
+			if (DL_ERROR_OK != err) return err;
 			dl_binary_writer_write_string_fmt(writer, "DL_ALIGN(8) %s", enum_info.name);
-			return;
+		    return DL_ERROR_OK;
 		}
-		case DL_TYPE_STORAGE_INT8:   dl_binary_writer_write_string_fmt(writer, "int8_t"); return;
-		case DL_TYPE_STORAGE_UINT8:  dl_binary_writer_write_string_fmt(writer, "uint8_t"); return;
-		case DL_TYPE_STORAGE_INT16:  dl_binary_writer_write_string_fmt(writer, "int16_t"); return;
-		case DL_TYPE_STORAGE_UINT16: dl_binary_writer_write_string_fmt(writer, "uint16_t"); return;
-		case DL_TYPE_STORAGE_INT32:  dl_binary_writer_write_string_fmt(writer, "int32_t"); return;
-		case DL_TYPE_STORAGE_UINT32: dl_binary_writer_write_string_fmt(writer, "uint32_t"); return;
-		case DL_TYPE_STORAGE_INT64:  dl_binary_writer_write_string_fmt(writer, "DL_ALIGN(8) int64_t"); return;
-		case DL_TYPE_STORAGE_UINT64: dl_binary_writer_write_string_fmt(writer, "DL_ALIGN(8) uint64_t"); return;
-		case DL_TYPE_STORAGE_FP32:   dl_binary_writer_write_string_fmt(writer, "float"); return;
-		case DL_TYPE_STORAGE_FP64:   dl_binary_writer_write_string_fmt(writer, "DL_ALIGN(8) double"); return;
-		case DL_TYPE_STORAGE_STR:    dl_binary_writer_write_string_fmt(writer, "const char*"); return;
-		case DL_TYPE_STORAGE_PTR:
+		case DL_TYPE_STORAGE_INT8:     dl_binary_writer_write_string_fmt(writer, "int8_t"); return DL_ERROR_OK;
+		case DL_TYPE_STORAGE_UINT8:    dl_binary_writer_write_string_fmt(writer, "uint8_t"); return DL_ERROR_OK;
+		case DL_TYPE_STORAGE_INT16:    dl_binary_writer_write_string_fmt(writer, "int16_t"); return DL_ERROR_OK;
+		case DL_TYPE_STORAGE_UINT16:   dl_binary_writer_write_string_fmt(writer, "uint16_t"); return DL_ERROR_OK;
+		case DL_TYPE_STORAGE_INT32:    dl_binary_writer_write_string_fmt(writer, "int32_t"); return DL_ERROR_OK;
+		case DL_TYPE_STORAGE_UINT32:   dl_binary_writer_write_string_fmt(writer, "uint32_t"); return DL_ERROR_OK;
+		case DL_TYPE_STORAGE_INT64:    dl_binary_writer_write_string_fmt(writer, "DL_ALIGN(8) int64_t"); return DL_ERROR_OK;
+		case DL_TYPE_STORAGE_UINT64:   dl_binary_writer_write_string_fmt(writer, "DL_ALIGN(8) uint64_t"); return DL_ERROR_OK;
+		case DL_TYPE_STORAGE_FP32:     dl_binary_writer_write_string_fmt(writer, "float"); return DL_ERROR_OK;
+		case DL_TYPE_STORAGE_FP64:     dl_binary_writer_write_string_fmt(writer, "DL_ALIGN(8) double"); return DL_ERROR_OK;
+		case DL_TYPE_STORAGE_STR:    dl_binary_writer_write_string_fmt(writer, "const char*"); return DL_ERROR_OK;
+	    case DL_TYPE_STORAGE_PTR:
```

Comment:
What happened here?

Reply:
Visual Studio wanted spaces instead of tabs...